lib/improver/tests/utilities/cube_manipulation/test_sort_coord_in_cube.py

```diff
@@ -49,34 +48,38 @@ class Test_sort_coord_in_cube(IrisTest):
     """Class to test the sort_coord_in_cube function."""
 
     def setUp(self):
-        """Set up a cube."""
-        self.ascending_height_points = np.array([5., 10., 20.])
-        cube = set_up_height_cube(self.ascending_height_points)[:, 0, :, :, :]
-        data = np.zeros(cube.shape)
-        data[0] = np.ones(cube[0].shape, dtype=np.int32)
-        data[1] = np.full(cube[1].shape, 2, dtype=np.int32)
-        data[2] = np.full(cube[2].shape, 3, dtype=np.int32)
-        cube.data = data
-        self.ascending_cube = cube
-        descending_cube = cube.copy()
-        self.descending_height_points = np.array([20., 10., 5.])
-        descending_cube.coord("height").points = self.descending_height_points
-        self.descending_cube = descending_cube
+        """Set up ascending and descending cubes"""
+        self.ascending_height_points = np.array(
+            [5., 10., 20.], dtype=np.float32)
+        self.descending_height_points = np.flip(self.ascending_height_points)
+
+        data = np.array(
+            [np.ones((3, 3)), 2*np.ones((3, 3)), 3*np.ones((3, 3))],
+            dtype=np.float32)
+        self.ascending_cube = set_up_variable_cube(data)
+        self.ascending_cube.coord("realization").rename("height")
+        self.ascending_cube.coord("height").points = (
+            self.ascending_height_points)
+        self.ascending_cube.coord("height").units = "m"
+
+        self.descending_cube = self.ascending_cube.copy()
+        self.descending_cube.coord("height").points = (
+            self.descending_height_points)
 
     def test_ascending_then_ascending(self):
         """Test that the sorting successfully sorts the cube based
         on the points within the given coordinate. The points in the resulting
         cube should now be in ascending order."""
         expected_data = np.array(
-            [[[[1.00, 1.00, 1.00],
-               [1.00, 1.00, 1.00],
-               [1.00, 1.00, 1.00]]],
-             [[[2.00, 2.00, 2.00],
-               [2.00, 2.00, 2.00],
-               [2.00, 2.00, 2.00]]],
-             [[[3.00, 3.00, 3.00],
-               [3.00, 3.00, 3.00],
-               [3.00, 3.00, 3.00]]]])
+            [[[1.00, 1.00, 1.00],
+              [1.00, 1.00, 1.00],
+              [1.00, 1.00, 1.00]],
+             [[2.00, 2.00, 2.00],
+              [2.00, 2.00, 2.00],
+              [2.00, 2.00, 2.00]],
+             [[3.00, 3.00, 3.00],
+              [3.00, 3.00, 3.00],
+              [3.00, 3.00, 3.00]]])
```

Comment:
I don't know if this makes it opaque, but this can be done in one line:
```
np.broadcast_to(np.arange(1, 4), (3, 3, 3)).T
```
Yeah, that's pretty opaque. Best leave it long form (I'll leave this here for interest).

Reply:
Oh but it's pretty!  Could compromise with `np.broadcast_to([1, 2, 3], (3, 3, 3)).T`, I think that's clearer.

Reply:
Especially if I set it up in the same way above...